### PR TITLE
Minutes/20220308

### PR DESCRIPTION
**Pull request by florcabral**
Adding the SWG meeting minutes for March 08, 2022 meeting.

### CODE REVIEW

File: meetings/2022_03_08/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- (Florencia) Presentation "NoDL and its relation to sros2"
+ 
+## Attendees
+ 
+Tomoya Fujita - Sony
+Gianluca Caiazza
+Guillaume Beuzeboc
+Roger Strain
+Marco Gutierrez
+Victor Mayoral-Vilches
```
Comment (vmayoral) [inline meetings/2022_03_08/README.md]:
```suggestion
Victor Mayoral-Vilches | Alias Robotics
```

**Comment by vmayoral** [comment]
Looking great @florcabral! Thanks a lot for managing this!

**Review by florcabral — COMMENTED** [review]
Change approved

**Review by artivis — APPROVED** [review]

**Comment by artivis** [comment]
Rebased for merging.

**Pull request merged**